spec/features/test_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+feature 'Testing capybara', type: :feature, vcr: true do
```

Comment:
what is the purpose of this test?

Reply:
renaming this of removing it would be good.

Reply:
this was created just as a template to get started.  I will remove it